README.md

```diff
@@ -50,6 +50,9 @@ This panel allows you to see all the rules you set at once. You can modify or de
 
 Sometimes it is not possible to select an application in the *Rules Editor*, especially for Steam games. This panel lets you set a rule for any running application. Of course the rules you set in this panel will be available directly in the *Rules Editor*. You can also remove a rule by setting the application behaviour to <img src="https://github.com/Pyroh/Fluor/blob/master/Fluor/Assets.xcassets/defaultModeTemplate.imageset/defaultModeTemplate@2x.png?raw=true" width=16pt>, it will also disappear from the *Rules Editor* panel.
 
+## Support for Touch Bar
+Macbook has a built in support for Touch Bar settings - Follow [this guide](https://www.techrepublic.com/article/how-to-permanently-display-the-function-keys-for-certain-apps-in-the-macbook-pro-touch-bar) to set it up for your applications
```

Comment:
It's best practice to end an sentence with a period. 😉 
Add it and I'll merge the PR.

Thank you.

Reply:
@Pyroh Sure thing. Added the dot. :)